README.md

```diff
@@ -1 +1,114 @@
+
+### Back-End Repository
+
+[Motorbike Booking Back-end](https://github.com/RWUBAKWANAYO/motorbike_booking_backend)
+
+### Kanban Board
+[Kanban Board](https://github.com/RWUBAKWANAYO/motorbike_booking_frontend/projects/1)
+
+> ***We are a group of five***
+1. Member 1 - [Pedro](https://github.com/ppmarq1)
+2. Member 2 - [Joseph](https://github.com/Kwentsir)
+3. Member 3 - [Emmanuel](https://github.com/Epaltechs)
+4. Member 4 - [Ghilain](https://github.com/Ghilain)
+5. Member 5 - [Olivier](https://github.com/RWUBAKWANAYO)
+
+## Built With
+
+- Major languages (Javascript)
+- Framworks (React, Redux)
+
+## Live version
+
+[https://motorbike-booking.netlify.app/]
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- A text editor(preferably Visual Studio Code)
+
+### Install
+- React
+- Redux
+
+### Using it Locally
+
+- Clone the project
+```
+git clone https://github.com/RWUBAKWANAYO/motorbike_booking_frontend
+cd motorbike_booking_frontend
+```
+
+### Setup
+
+Start server with:
+
+```
+npm install
+```
+```
+npm start
+```
+
+Visit http://localhost:3000/ in your browser.
+
+## Visit And Open Files
+
+[Visit Repo](https://github.com/RWUBAKWANAYO/motorbike_booking_frontend)
+
+
+## Authors
+
+👤 **Olivier**
+
+- GitHub: [@Rwubakwanayo](https://github.com/RWUBAKWANAYO)
+- Twitter: [@Rwubakwanayo](https://twitter.com/rwubakwanayo
+)
+- LinkedIn: [Rwubakwanayo Oliver](https://www.linkedin.com/in/rwubakwanayo-olivier)
+
+👤 **Pedro**
+
+- GitHub: [Pedro](https://github.com/ppmarq1)
+- Twitter: [@ppmarq1](https://twitter.com/ppmarq1)
+- LinkedIn: [@Pedro](https://br.linkedin.com/in/pedroalmeidamarques/)
+
+👤 **Joseph**
+
+- GitHub: [@Kwentsir](https://github.com/Kwentsir/)
+- Twitter: [@jkwentsir](https://twitter.com/jkwentsir)
+- LinkedIn: [@Joseph Kwentsir](https://www.linkedin.com/in/josephkwentsir/)
+
+👤 **Ghilain**
+
+- GitHub: [@Ghilain Ishimwe](https://github.com/Ghilain)
+- Twitter: [@GhilainIshime](https://twitter.com/GhilainIshimwe)
+- LinkedIn: [@Ghilain Ishimwe](https://www.linkedin.com/in/ghilain-ishimwe/)
+
+👤 **Emmanuel**
+
+- GitHub: [@Epaltechs](https://github.com/Epaltechs)
+- Twitter: [Emmanuel Paul](http://twitter.com/@emmapaul247)
+- LinkedIn: [Emmanuel Paul](https://www.linkedin.com/in/emmanuel-s-paul)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/RWUBAKWANAYO/motorbike_booking_frontend/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+-  We acknowledge with appreciation the original design provided by Murat Korkmaz on Behance.
+- Inspiration: Microverse
+
+## 📝 License
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ ] This link does not work because it points to a file that does not exist.  Please, add a license file to the root of your project and link to it here on your readme.   You can grab the same license file from your backend project 💪